getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -946,13 +946,54 @@ The class constructor, called on class instantiation, is named ``_init``.
 As mentioned earlier, the constructors of parent classes are called automatically when
 inheriting a class. So there is usually no need to call ``._init()`` explicitly.
 
-If a parent constructor takes arguments, they are passed like this:
+Unlike the call of a regular function like in the above example with ``.some_func``,
+if the constructor from the inherited class takes arguments, they are passed like this:
 
 ::
 
     func _init(args).(parent_args):
        pass
 
+This is better explained through examples. Say we have this scenario:
+
+::
+
+    # State.gd (inherited class)
+    var entity = null
+    var message = null
+
+    func _init(e=null):
+        entity = e
+
+    func enter(m):
+        message = m
+
+
+    # Idle.gd (inheriting class)
+    extends "State.gd"
+
+    func _init(e=null, m).(e):
```

Comment:
If you add a default value to an argument, the next arguments also need a default value.

Reply:
Indeed! Nice catch